src/utils/cardUtilityFunctions.js

```diff
@@ -2,6 +2,8 @@ import warning from 'warning';
 
 import { CARD_SIZES } from '../constants/LayoutConstants';
 
+/* eslint-disable no-unused-expressions */
```

Comment:
This is disabling the entire file and can probably be fixed. Not sure which expressions are being unused though

Reply:
The linter doesn't like this syntax: 

thresholds?.forEach()

I think it expects some sort of 'else' statement. I can change to only disable those specific instances instead.

Reply:
Go ahead and leave it since the rule hasn't been updated to understand that syntax

Reply:
We can isolate the disable to a specific line via one of the following

```
alert('foo'); // eslint-disable-line no-alert

// eslint-disable-next-line no-alert
alert('foo');

alert('foo'); /* eslint-disable-line no-alert */

/* eslint-disable-next-line no-alert */
alert('foo');
```

Reply:
I can change to inline, although there are 21 instances of this in that file. Seemed cleaner to disable the whole file. Thoughts @tay1orjones ?